src/ussd_tb_check.js

```diff
@@ -14,6 +14,14 @@ go.app = (function () {
     App.call(self, "state_start");
     var $ = self.$;
 
+    self.languages = {
+      zul: "isiZulu",
+      xho: "isiXhosa",
+      afr: "Afrikaans",
+      eng: "English",
+      sot: "Sesotho"
+  };
```

Comment:
It doesn't seem like we're using this anywhere?

Reply:
ahh, I thought this was needed for the testing setup. Will remove